release-notes/opensearch.release-notes-2.4.1.md

```diff
@@ -0,0 +1,22 @@
+## 2022-12-07 Version 2.4.1 Release Notes
```

Comment:
Hm ... it is a right date btw?

Reply:
Yes, this needed to be updated. Raised a manual backport PR: https://github.com/opensearch-project/OpenSearch/pull/5550